network-api/networkapi/mozfest/templates/partials/streamfield.html

```diff
@@ -1,9 +1,19 @@
 {% load wagtailcore_tags %}
 
 {% for block in page.body %}
-  {% if block.block_type == 'heading' %}
-    <h1>{{ block.value }}</h1>
-  {% else %}
-    {% include_block block %}
-  {% endif %}
+{% if block.block_type == 'paragraph'%}
+<div class="container {{block.block_type}}-block">
+  <div class="row justify-content-center">
+    <div class="streamfield-content">
+      {% if block.block_type == 'heading' %}
```

Comment:
These `if block.block_type == 'heading'` conditionals won't execute because the parent condition is too strict. But let's check if there's a "heading" block type, and if there is we'll want to support it still.

Reply:
Hi Kalob, I looked through the pages that are checking for 'heading' and can confirm that 'heading' is not an option when selecting a new streamfield. Only 'paragraph' or 'content', however, I am going to make a ticket to create a new "paragraph" streamfield block, that way instead of using a conditional, we can just loop through all of the blocks.

Reply:
I resolved the 'heading' conditional, and have created ticket [#6920](https://github.com/mozilla/foundation.mozilla.org/issues/6920) to create a new paragraph block, that way we can loop through all blocks instead of having a conditional for paragraphs!